engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -4566,6 +4589,13 @@ struct soul_cleave_t : public demon_hunter_attack_t
       }
     }
 
+    void execute() override
+    {
+      demon_hunter_attack_t::execute();
+
+      p()->buff.soul_furnace_damage_amp->expire();
+    }
+
     double composite_da_multiplier( const action_state_t* s ) const override
     {
       double m = demon_hunter_action_t::composite_da_multiplier( s );
```

Comment:
Same issue here actually. I just missed it in the previous review for Focused Cleave.